packages/fast-usdc-contract/src/exos/advancer.ts

```diff
@@ -145,6 +150,16 @@ export const prepareAdvancerKit = (
   const feeTools = makeFeeTools(feeConfig);
   const toAmount = (value: bigint) => AmountMath.make(usdc.brand, value);
 
+  // UNTIL we can put this in the Exo state https://github.com/Agoric/agoric-sdk/issues/10200
+  const getIntermediaryAccount = () => {
+    const intermediaryAccount = zone.makeOnce(
+      INTERMEDIARY_ACCOUNT,
+      () => Fail`Intermediary account not set`,
+    );
+    assert(intermediaryAccount);
+    return intermediaryAccount;
+  };
```

Comment:
`IntermediaryAccount` is the same as `NOBLE_ICA`, so maybe we should pass in `getNobleICA()` like we do for `Settler`

Reply:
done